Spec/Auth.swift

```diff
@@ -3503,13 +3466,11 @@ override class var defaultTestSuite : XCTestSuite {
                         }
                     }
                 }
-
-            }
             
             // RSA10k
-            context("server time offset") {
+            
 
-                xit("should obtain server time once and persist the offset from the local clock") {
+                func skipped__test__115__authorize__server_time_offset__should_obtain_server_time_once_and_persist_the_offset_from_the_local_clock() {
```

Comment:
Notice that `xit` gets turned into a `skipped__test` method. What do you think about this? I know there also exists [`XCTSkip`](https://developer.apple.com/documentation/xctest/xctskip), an error which when thrown will skip the test case and I think mark it as skipped in the Xcode test navigator. Do you think it's worth me investigating or is this fine?

Reply:
It's fine IMHO